content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -27,9 +27,35 @@ repo here](https://github.com/kubernetes/website/tree/{{< param "docsbranch" >}}
 
 {{% capture body %}}
 
+## nodeName
+
+`nodeName` is the simplest node selection constraint.
+`nodeName` is a field of PodSpec.  If it is non-empty, the scheduler simply schedules this pod onto that node,
+assuming that it fits resource requirements.  Since resource requirements are ignored, this approach for node
```

Comment:
Resource requirements are not ignored. As you pointed in the next sentence, if the specified node does not have enough resources, the kubelet will reject the pod and the pod gets deleted.

Reply:
Fixed